x-pack/filebeat/input/cel/input_test.go

```diff
@@ -31,594 +31,655 @@ import (
 	"github.com/elastic/elastic-agent-libs/mapstr"
 )
 
-var inputTests = []struct {
```

Comment:
I didn't notice this (hidden for GitHub folding changes). It should not have been changed; large change with no reason.